csi/node_server.go

```diff
@@ -899,6 +860,86 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	return &csi.NodeExpandVolumeResponse{CapacityBytes: requestedSize}, nil
 }
 
+// prepareAndExpandEncryptedVolume handles the expansion of encrypted volumes and
+// returns volume is a block volume, updated device path, and error if any.
+func (ns *NodeServer) prepareAndExpandEncryptedVolume(req *csi.NodeExpandVolumeRequest, volume *longhornclient.Volume, volumeID, devicePath string) (bool, string, error) {
+	isBlockVolume := req.VolumeCapability.GetBlock() != nil
+	if isBlockVolume {
+		log.Debugf("Volume %v on node %v does not require filesystem resize since it is access mode Block", volumeID, ns.nodeID)
+	}
```

Comment:
Move 
```
isBlockVolume := req.VolumeCapability.GetBlock() != nil
```
outside the function, and isBlockVolume is one th parameters of this function.